### PR TITLE
the met

### PR DESCRIPTION
**Pull request by VladyslavKolisnyk**
DEMO LINK https://vladyslavkolisnyk.github.io/layout_landing-page

### CODE REVIEW

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good work 👍 
Let's to fix some moments
Add transition and cursor: pointer for all clickable elements when on hover
image
image

Need to remove default styles after autocomplete
image

**Review by loralevitska — CHANGES_REQUESTED** [review]
Good job! There are things that should be fixed:

- still don't fixed 
image
- disable text-area resize, it breaks design
image
- also add cursor pointer to all interactive elements

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Check indents on mobile version
image

Add additional styles for these elements when hovering
image

File: src/index.html
Comment (anastasiiavorobiova) [inline src/index.html:1]: Check the validity of your HTML with [HTML validator](https://validator.w3.org/nu/#textarea)

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Really nice looking landing! Let's make it perfect

**Review by anastasiiavorobiova — APPROVED** [review]
Awesome!

**Pull request closed without merging**